readthedocs/gold/views.py

```diff
@@ -140,3 +141,47 @@ def thanks(request):
         },
         context_instance=RequestContext(request)
     )
+
+
+@login_required
+def projects(request):
+    try:
+        gold_user = GoldUser.objects.get(user=request.user)
+        gold_projects = gold_user.projects.all()
+    except:
+        raise Http404('Gold User not found')
+
+    if request.method == 'POST':
+        form = GoldProjectForm(request.POST)
+        if form.is_valid():
+            if gold_projects.count() < gold_user.num_supported_projects:
```

Comment:
That is validation logic and should live in the form. Otherwise you will need to duplicate in every place the form gets used in the future.